pkg/scheduler/cache/cache.go

```diff
@@ -768,24 +767,22 @@ func (sc *SchedulerCache) Evict(taskInfo *schedulingapi.TaskInfo, reason string)
 }
 
 // Bind binds task to the target host.
-func (sc *SchedulerCache) Bind(tasks []*schedulingapi.TaskInfo) error {
-	go func(taskArray []*schedulingapi.TaskInfo) {
-		tmp := time.Now()
-		errTasks, err := sc.Binder.Bind(sc.kubeClient, taskArray)
-		if err == nil {
-			klog.V(3).Infof("bind ok, latency %v", time.Since(tmp))
-			for _, task := range tasks {
-				sc.Recorder.Eventf(task.Pod, v1.EventTypeNormal, "Scheduled", "Successfully assigned %v/%v to %v",
-					task.Namespace, task.Name, task.NodeName)
-			}
-		} else {
-			for _, task := range errTasks {
-				klog.V(2).Infof("resyncTask task %s", task.Name)
-				sc.VolumeBinder.RevertVolumes(task, task.PodVolumes)
-				sc.resyncTask(task)
-			}
+func (sc *SchedulerCache) Bind(taskArray []*schedulingapi.TaskInfo) error {
```

Comment:
What is the reason for Volcano's original asynchronous binding, performance?
Does the modification to synchronous have any impact on performance?

Reply:
ok,here The reason which  change asynchronous  to synchronous    is   I have move the asynchronous   to above little , so here is not need